### PR TITLE
Need more detail for contributing to v3 web console, started an architecture section

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
@liggitt review

@jcantrill @csrwng @fabianofranz  FYI.  As you add more directives and filters to our source, make sure to add them to this list if you think they are worth noting.  Things that were less re-usable I left off the list.

File: assets/README.md [hunk 1]
```diff
@@ -38,3 +38,51 @@ If Travis complains that bindata.go is different than the committed version, ens
...
+
+The OpenShift v3 management console is based on AngularJS and [Hawt.io](https://github.com/hawtio/hawtio-core)
+
+#### Custom directives and filters
+
+The v3 console relies heavily on custom directives and filters, some of which are intended to be utilties and used throughout the console source. The list below is NOT a complete list of all of our directives and filters.
+
+##### Directives
+
+For more details on the expected scope arguments, see the source under [app/scripts/directives](https://github.com/openshift/origin/tree/master/assets/app/scripts/directives)
```
Comment (liggitt) [inline assets/README.md]: can the links be relative?

**Comment by liggitt** [comment]
LGTM

**Comment by jwforres** [comment]
[merge]

**Pull request merged**